desktop/core/src/desktop/settings.py

```diff
@@ -655,24 +655,11 @@ def is_oidc_configured():
 
 # Insert our custom upload handlers
 file_upload_handlers = [
-    'hadoop.fs.upload.HDFSfileUploadHandler',
+    'hadoop.fs.upload.FineUploaderChunkedUploadHandler',
     'django.core.files.uploadhandler.MemoryFileUploadHandler',
     'django.core.files.uploadhandler.TemporaryFileUploadHandler',
 ]
 
-if is_s3_enabled():
-  file_upload_handlers.insert(0, 'aws.s3.upload.S3FileUploadHandler')
-
-if is_gs_enabled():
-  file_upload_handlers.insert(0, 'desktop.lib.fs.gc.upload.GSFileUploadHandler')
-
-if is_abfs_enabled():
-  file_upload_handlers.insert(0, 'azure.abfs.upload.ABFSFileUploadHandler')
-
-if is_ofs_enabled():
-  file_upload_handlers.insert(0, 'desktop.lib.fs.ozone.upload.OFSFileUploadHandler')
-
-
```

Comment:
How will the users fall back to the old implementation of file upload if the configs is made True?
Aren't we just deletion the connections of old Django's own file upload handler implementation here?